cypress/integration/lpa/donors/donor.spec.js

```diff
@@ -1,35 +1,44 @@
-// describe("Create Donor", { tags: ["@lpa", "@smoke-journey"] }, () => {
-//   beforeEach(() => {
-//     cy.loginAs("Case Manager");
-//     cy.visit("/lpa/home");
-//   });
-
-//   it("should create a new donor", function () {
-//     cy.intercept({ method: "GET", url: "/*/v1/assignees/*/tasks*" }).as(
-//       "tasksRequest"
-//     );
-//     cy.wait("@tasksRequest");
-
-//     cy.get("uib-tab-heading[id=Timeline]").contains("Timeline").click();
-//     cy.contains("Create Donor").click();
-
-//     cy.frameLoaded(".action-widget-content iframe");
-//     cy.enter(".action-widget-content iframe").then((getBody) => {
-//       getBody().find("#f-firstname").type("Spongebob");
-//       getBody().find("#f-surname").type("Squarepants");
-//       getBody().find("button[type=submit]").click();
-
-//       getBody().contains(/Person (\d+(-|)){3} was created/);
-//       getBody().contains("View donor").click();
-//     });
-
-//     cy.get(".timeline .timeline-event", { timeout: 10000 });
-//     cy.contains(".timeline-event", "Person (Create / Edit)").should(
-//       "contain",
-//       "Spongebob Squarepants"
-//     );
-//   });
-// });
+describe("Create Donor", { tags: ["@lpa", "@smoke-journey"] }, () => {
+  beforeEach(() => {
+    cy.loginAs("Case Manager");
+    cy.visit("/lpa/home");
+  });
+
+  it("should create a new donor", function () {
+    cy.intercept({ method: "GET", url: "/*/v1/assignees/*/tasks*" }).as(
+      "tasksRequest"
+    );
+    cy.wait("@tasksRequest");
+
+    cy.get("uib-tab-heading[id=Timeline]").contains("Timeline").click();
+    cy.contains("Create Donor").click();
+
+    cy.frameLoaded(".action-widget-content iframe");
+    cy.enter(".action-widget-content iframe").then((getBody) => {
+      getBody().find("#f-firstname").type("Spongebob");
+      getBody().find("#f-surname").type("Squarepants");
+      getBody().find("button[type=submit]").click();
+    });
+
+    cy.wait(1000);
+    cy.enter(".action-widget-content iframe").then((getBody) => {
+      getBody().contains(/Person (\d+(-|)){3} was created/);
+
+      getBody()
+        .contains("a", "View donor")
+        .invoke("attr", "href")
+        .then((donorLinkUrl) => {
+          cy.visit(donorLinkUrl.replace(/https?:\/\/localhost:8080/, Cypress.config('baseUrl')));
```

Comment:
This sucks, but it avoids a cross-domain problem because the site is hosted at `frontend-proxy` but generates links as `localhost:8080`. I expect that can be fixed but it would involve coordination beyond this PR.

Also we can't click the link because it opens in `_top`, which would break out of the _Cypress_ iframe.

As with a lot of the weirdness in these tests, they'll be fixed when we stop needing to use iframes.